indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Encoding.kt

```diff
@@ -518,3 +510,48 @@ fun ByteArray.ensureSize(size: Int): ByteArray = (this.size - size).let { toDrop
 
 @Suppress("NOTHING_TO_INLINE")
 inline fun ByteArray.ensureSize(size: UInt) = ensureSize(size.toInt())
+
+/**
+ * Encodes this number using varint encoding as used within ASN.1: groups of seven bits are encoded into a byte,
+ * while the highest bit indicates if more bytes are to come
+ */
+fun ULong.toAsn1VarInt(): ByteArray {
+    if (this < 128u) return byteArrayOf(this.toByte()) //Fast case
+    var offset = 0
+    var result= mutableListOf<Byte>()
```

Comment:
Could use automatic reformatting

Reply:
do me a favour, please and try auto-format with this project on your machine. it is simply not working for me